src/components/PointsBar.js

```diff
@@ -19,7 +19,7 @@ function PointsBar({ user }) {
                 </Segment>
                 <Segment vertical>
                   <p className="points-header">
-                    {user ? user.fallPercentile : "0"} percentile
+                    Top {user ? user.fallPercent : "0"} percent
```

Comment:
Do we want it to show as, for example, top 95% or top 5%. I don't remember exaclty what we talked about, so do whichever Fernando specified before!